src/sqswatcher/plugins/sge.py

```diff
@@ -99,5 +103,52 @@ def update_cluster(max_cluster_size, cluster_user, update_events, instance_prope
     return failed, succeeded
 
 
+def _is_node_locked(hostname):
+    try:
+        node_info = get_compute_nodes_info(hostname_filter=hostname)
+        node = node_info.get(socket.getfqdn(hostname), node_info.get(hostname))
+        if SGE_DISABLED_STATE in node.state:
+            return True
+    except Exception as e:
+        log.error(
+            "Failed when checking if node is locked with exception %s. Reporting node %s as unlocked.", e, hostname
+        )
+    return False
+
+
+def perform_health_actions(health_events):
+    """Update and write node lists( and gres_nodes if instance has GPU); restart relevant nodes."""
+    failed = []
+    succeeded = []
+    for event in health_events:
+        try:
+            if _is_node_locked(event.host.hostname):
+                log.warning(
+                    "Instance %s/%s currently in disabled state 'd'. "
```

Comment:
in disabled state 'd'.  --> in the message we should use the SGE_DISABLED_STATE constant defined before.

Reply:
Done